components/breadcrumb/src/Crumb.js

```diff
@@ -67,7 +67,9 @@ Crumb.propTypes = {
 	current: PropTypes.bool,
 
 	/**
-	 * An href for a link
+	 * The crumb link href value.
+	 *
+	 * Defaults to '#0'.
```

Comment:
What do you mean "defaults to '#0'? There is no defaultProp declared tho

Reply:
Defaulted in code